assignments.js

```diff
@@ -0,0 +1,53 @@
+/* Fibonacci series using map Assignment question-1 */
+
+export function fibonacciSeries(number) {
+  const fib = [0, 1];
+  for (let i = 2; i <= number; i++) {
+    fib[i] = fib[i - 1] + fib[i - 2];
+  }
+  return fib;
+}
+export function mappingIndex(numbers) {

```

Comment:
This function name is very confusing. Imagine someone else now wants to make use of this. basically we want `fibNumbersAtIndices`. This name is apt here